docs/UserGuide.md

```diff
@@ -399,72 +400,119 @@ Examples:
 
 For undoing a clone command:
 
-* Clone a contact.
+* Clone a contact
+
+![undo format](images/undo-UG/after_clone_1.png)
 
-![edit format](images/undo-UG/after_clone_1_scrolled_down.png)
+* Undo
+* Message shown to the user: 
 
-* Undo.
-* Message shown to the user: "Undo Successful! Deleted Person: X", where X are the details of the person who was
-  just cloned, and now deleted.
-* GUI reflects that the most recently cloned contact is deleted.
+```
+Undo Successful! Deleted Person: X
+```
+where X are the details of the person who was just cloned, and now deleted
+* GUI reflects that the most recently cloned contact is deleted
 
-![edit format](images/undo-UG/after_clone_undo_scrolled_down.png)
+![undo format](images/undo-UG/after_clone_undo.png)
 
 For undoing a delete command:
 
-* Delete a contact.
+* Delete a contact
 
-![edit format](images/undo-UG/after_delete_1.png)
+![undo format](images/undo-UG/after_delete_1.png)
 
-* Undo.
-* Message shown to the user: "Undo Successful! Contact added back: X", where X are the details of the person who was
-  just deleted, and now added back.
-* GUI reflects that the most recently deleted contact is added back.
+* Undo
+* Message shown to the user: 
 
-![edit format](images/undo-UG/after_delete_undo_scrolled_down.png)
+```
+Undo Successful! Contact(s) added back: X
+```
+where X are the details of the person who was just deleted, and now added back
+* GUI reflects that the most recently deleted contact is added back
+
+![undo format](images/undo-UG/after_delete_undo.png)
+
+* Delete multiple contacts (In this case, 2 contacts)
+
+![undo format](images/undo-UG/after_delete_multiple.png)
+
+* Undo
+* Message shown to the user: 
+
+```
+Undo Successful! Contact(s) added back: X
+```
+where X are the details of the persons who were just deleted, and now added back
+* GUI reflects that the most recently deleted contacts are added back
+
+![undo_format](images/undo-UG/after_delete_multiple_undo.png)
 
 For undoing a clear command:
 
-* Clear all contacts.
+* Clear all contacts
 
 ![undo format](images/undo-UG/after_clear.png)
 
-* Undo.
-* Message shown to the user: "Undo Successful! ALl contacts have been added back!".
-* GUI reflects that all cleared contacts are added back.
+* Undo
+* Message shown to the user: 
+
+```
+Undo Successful! All contacts have been added back!
+```
+
+* GUI reflects that all cleared contacts are added back
 
 ![undo format](images/undo-UG/after_clear_undo.png)
 
 For undoing an edit command:
 
-* Edit a contact.
+* Edit a contact
 
-![undo format](images/undo-UG/after_edit_1.png)
 
-* Undo.
-* Message shown to the user: "Undo Successful! Reverted back to: X", where X are the details of the person before
-  the edit.
-* GUI reflects that the edited contact has been reverted.
+![undo format](images/undo-UG/after_edit.png)
+
+
+* Undo
+* Message shown to the user: 
+
+```
+Undo Successful! Reverted back to: X
+```
+
+where X are the details of the person before
+  the edit
```

Comment:
Is "the edit" being on the new line intentional? Not that this format is wrong, just checking!

Reply:
Ah, will reformat this, thanks!